day_6/exercises/burrito.rb

```diff
@@ -18,11 +18,13 @@ def add_topping(new_topping)
   end
 
   def remove_topping
-    puts "Please remove the #{toppings[0]}, many thanks!"
+    puts "Please remove the #{toppings[0]} from the menu, many thanks!"
+    toppings.shift
   end
 
-  def change_protein
-    puts "Can I please change my protein from #{protein} to Chicken, thanks!"
+  def change_protein(new_protein)
+    protein.sub("Beans", new_protein)
```

Comment:
Interesting approach. I probably would have reassigned with a simple `@protein = new_protein` (using `@` to reference the instance variable in this class.